recipes/repo.rb

```diff
@@ -17,6 +17,8 @@
 # limitations under the License.
 #
 
+repo_version = node['cdap']['version']}.to_f
```

Comment:
Remove `}`...

This is failing the Travis test... https://travis-ci.org/caskdata/cdap_cookbook/builds/48934465#L192-L193